goship.go

```diff
@@ -28,16 +28,26 @@ import (
 	"github.com/coreos/go-etcd/etcd"
 	"github.com/gengo/goship/lib"
 	"github.com/google/go-github/github"
+	"github.com/gorilla/sessions"
+	"github.com/stretchr/gomniauth"
+	githubOauth "github.com/stretchr/gomniauth/providers/github"
+	"github.com/stretchr/objx"
 )
 
 var (
-	bindAddress = flag.String("b", "localhost:8000", "Address to bind (default localhost:8000)")
-	sshPort     = "22"
-	keyPath     = flag.String("k", "id_rsa", "Path to private SSH key (default id_rsa)")
-	dataPath    = flag.String("d", "data/", "Path to data directory (default ./data/)")
-	ETCDServer  = flag.String("e", "http://127.0.0.1:4001", "Etcd Server (default http://127.0.0.1:4001")
+	bindAddress       = flag.String("b", "localhost:8000", "Address to bind (default localhost:8000)")
+	sshPort           = "22"
+	keyPath           = flag.String("k", "id_rsa", "Path to private SSH key (default id_rsa)")
+	dataPath          = flag.String("d", "data/", "Path to data directory (default ./data/)")
+	ETCDServer        = flag.String("e", "http://127.0.0.1:4001", "Etcd Server (default http://127.0.0.1:4001)")
+	cookieSessionHash = flag.String("c", "jhjhjhjhjhjjhjhhj", "Random cookie session key (default jhjhjhjhjhjjhjhhj)")
```

Comment:
Can we not make the default `jhjhjhjhjhjjhjhhj` please :smile_cat: 

Maybe something in all-caps "COOKIE-SESSION-HASH"